content/docs/platform/developer/webhooks/inbound-webhooks.mdx

```diff
@@ -0,0 +1,124 @@
+---
+pageTitle: 'Inbound Webhooks'
+title: 'Inbound Webhooks'
+description: 'Learn how to configure inbound webhooks to receive real-time notification statuses from third-party providers.'
+icon: 'Download'
+---
+
+Inbound webhooks let Novu receive and process real-time events sent by third-party providers such as SendGrid, Amazon SES, Mailgun, and Resend. These events describe what happens to notifications after they are sent by Novu, such as, when an email is delivered, opened, clicked, or bounced.
+
+When Novu receives an inbound webhook event, it:
+- Parses and normalizes the provider’s event into Novu’s internal status model.
+- Updates the message entity and step-run trace, so the event is visible in the activity feed.
+- Forwards supported events to outbound webhooks if they are enabled in your project.
+- Enriches the notification lifecycle with engagement and delivery data, which can also power analytics and workflow variables.
```

Comment:
i think we can make this simpler for the end user. they don’t need to know the internal details of Novu, they’ll be more interested in understanding how it affects them.

```suggestion
- Create step-run trace, so the event is visible in the activity feed.
- Forwards supported events to outbound webhooks if they are enabled in your project.
- Enriches the workflow-run lifecycle with engagement and delivery data, which empowers analytics and workflow variables.
```